src/js/gi/containers/SearchPage.jsx

```diff
@@ -101,7 +99,7 @@ export class SearchPage extends React.Component {
 
   handleFilterChange(field, value) {
     // Translate form selections to query params.
-    const queryValue = field === 'caution' ? !value : value;
+    const queryValue = value;
```

Comment:
I think we can just use `value` instead of assigning to `queryValue`; the only reason for reassigning was to conditionally set a value.